internal/test/e2e/kubectl_test.go

```diff
@@ -341,7 +342,15 @@ func eventuallyInCluster(kubectl *kubectlContext, name string, deadline time.Dur
 	if err != nil {
 		return "", err
 	}
-
+	defer func() {
+		err := kubectl.Delete("--filename", f.Name())
+		if err != nil {
+			reterr = kerrors.NewAggregate([]error{
+				reterr,
+				fmt.Errorf("error while deleting job in eventuallyInCluster: %s ", err),
+			})
+		}
+	}()
 	err = kubectl.Wait("job", name, "--for", fmt.Sprintf("condition=%s", batchv1.JobComplete), "--timeout", deadline.String())
 	if err != nil {
 		return "", err
```

Comment:
I renamed this file, because it references fFlagVariables which are defined in `e2e_test.go` ....which meant that `go build ./...` tried to compile this file in the absence of e2e_test.go (because of the _test suffix) 
I've made this file a `_test.go` file too.